.github/workflows/pr-preview.yml

```diff
@@ -62,11 +62,11 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.BOT }}
           message: |
-            ## Deploying with :page_facing_up: Github Pages
+            ## Deploying with :page_facing_up: GitHub Pages
```

Comment:
@WilliamDavidHarrison do you think we should change `Deploying` to `Deployed` here?